Wordle.Api/Wordle.Api/Services/WordEditorService.cs

```diff
@@ -0,0 +1,74 @@
+using Microsoft.EntityFrameworkCore;
+using Wordle.Api.Dtos;
+using Wordle.Api.Models;
+
+namespace Wordle.Api.Services;
+
+public class WordEditorService(WordleDbContext Db) 
+{
+    public async Task<bool> DeleteWordAsync(string word)
+    {
+        var wordToDelete = await Db.Words.FirstOrDefaultAsync(w => string.Equals(w.Text, word.ToLower()));
+        if (wordToDelete == null)
+        {
+            return false;
+        }
+
+        Db.Words.Remove(wordToDelete);
+        await Db.SaveChangesAsync();
+        return true;
```

Comment:
I am curious as to why you are returning true and false, might need to change wordeditorcontroller logic but this may not be needed.

Reply:
You are right. This was a lazy implementation and I am thinking of reworking this part to get more use out of this API endpoint.